python/hail/expr/dataset.py

```diff
@@ -0,0 +1,210 @@
+from __future__ import print_function  # Python 2 and 3 print compatibility
+
+import warnings
+
+from decorator import decorator
+from hail.expr.column import *
+
+
+warnings.filterwarnings(module=__name__, action='once')
```

Comment:
Are we getting a ton of warnings?  What for?

Reply:
I copied this from the old `dataset.py` file. @tpoterba did you add this line?

Reply:
Yeah, I added it. Specifically, in #1297. 

I did some archaeology -- I added it because I used warnings to warn about deprecation, but then killed the warnings without deleting the management line at the top. This can safely be removed.